File: testdata/projects/node/math.test.js
import { expect, test } from "vitest";
import { add, isPositive } from "./math.js";

test("add", () => {
  expect(add(2, 3)).toBe(5);
});

test("isPositive", () => {
  expect(isPositive(1)).toBe(true);
  expect(isPositive(-1)).toBe(false);
  expect(isPositive(0)).toBe(false);
});

File: testdata/projects/go/math_test.go
package math

import "testing"

func TestAdd(t *testing.T) {
	if Add(2, 3) != 5 {
		t.Error("expected 5")
	}
}

func TestIsPositive(t *testing.T) {
	if !IsPositive(1) {
		t.Error("expected true")
	}
	if IsPositive(-1) {
		t.Error("expected false")
	}
	if IsPositive(0) {
		t.Error("expected false")
	}
}
